src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -515,24 +528,6 @@ export default {
 			return this.getMessagesListScroller()
 		},
 
-		isModifiable() {
-			return !this.isConversationReadOnly && this.conversation.participantType !== PARTICIPANT.TYPE.GUEST
-		},
-
-		isOneToOne() {
-			return this.conversation.type === CONVERSATION.TYPE.ONE_TO_ONE
-				|| this.conversation.type === CONVERSATION.TYPE.ONE_TO_ONE_FORMER
-		},
-
-		isEditable() {
-			if (!canEditMessage || !this.isModifiable || this.isObjectShare
-					|| ((!this.$store.getters.isModerator || this.isOneToOne) && !this.isMyMsg)) {
-				return false
-			}
-
-			return (moment(this.timestamp * 1000).add(1, 'd')) > moment()
-		},
-
 		isDeleteable() {
```

Comment:
I'd consider move that to composable or at least simplify them with `isOneToOne`, `getters.isModerator` and so on